pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/testdata/retain-test.yaml

```diff
@@ -0,0 +1,200 @@
+# test case for retaining Workflow
+# case1. Workflow: retain test (recently completed)
+# case2. Workflow: retain test (old completed - TTL exceeded)
+# case3. Workflow retain should preserve observed labels
```

Comment:
I think keeping just this one case is enough; the other two cases can be removed.